modules/codegen/test/src/smithy4s/codegen/internals/RendererSpec.scala

```diff
@@ -484,4 +484,61 @@ final class RendererSpec extends munit.FunSuite {
       )
     )
   }
+
+  test("collision avoidance should cover `import`") {
+    val smithy =
+      """
+        |$version: "2"
+        |
+        |namespace input
+        |
+        |struct Import { import: String }
+        |""".stripMargin
+    val contents = generateScalaCode(smithy).values
+    assert(
+      contents.exists(
+        _.contains(
+          "case class Import(_import: Option[String] = None)"
+        )
+      )
+    )
+  }
+
+  test("collision avoidance should cover `export` for scala 3 compat") {
+    val smithy =
+      """
+        |$version: "2"
+        |
+        |namespace input
+        |
+        |struct Export { export: String }
+        |""".stripMargin
+    val contents = generateScalaCode(smithy).values
+    assert(
+      contents.exists(
+        _.contains(
+          "case class Export(_export: Option[String] = None)"
+        )
+      )
+    )
+  }
+
+  test("collision avoidance should cover `given` for scala 3 compat") {
+    val smithy =
+      """
+        |$version: "2"
+        |
+        |namespace input
+        |
+        |struct Given { given: String }
+        |""".stripMargin
+    val contents = generateScalaCode(smithy).values
+    assert(
+      contents.exists(
+        _.contains(
+          "case class Given(_given: Option[String] = None)"
+        )
+      )
+    )
+  }
```

Comment:
I was unable to get a working dev environment to run these tests, and took the gamble that they would compile and pass. If not or if changes are requested, I'll do the legwork to get tests running.